lib/configs.py

```diff
@@ -0,0 +1 @@
+AUTH_COMMAND = ''
```

Comment:
I would remove this file, no need to introduce this variable here?

Reply:
since this is global static variable being called by multiple modules/python files. so keeping it in separate module avoids cyclic dependency.

Reply:
I can keep it in `ClusterProvider.py`, I had gotten an issue earlier with this but will give it a try.

Reply:
I would say we keep this as a place to retrieve the config details at any time, if we want to avoid accessing the cluster. Or this could be a point to retrieve authentications for an existing(or just created) cluster.

Reply:
I would keep it in `ClusterProvider` until we see the need to access it from elsewhere.